spec/factories/users.rb

```diff
@@ -22,7 +22,7 @@
       verified_at { nil }
       sec_id { '123498767' }
       participant_id { Faker::Number.number(digits: 8) }
-      birls_id { Faker::Number.number(digits: 10) }
+      birls_id { Faker::Number.number(digits: 9) }
```

Comment:
This change seems fine, I think it's the only Identity related change.

Just curious, do you have a reference for birls id requirements, or a lighthouse contact I can just quickly ask about the 8-9 character limit? I'm just trying to figure out why this was set to 10 digits in the first place (my guess is there wasn't any reason, it was probably just randomly set to that)

Reply:
MPI data examples have it at 9 chars

https://github.com/department-of-veterans-affairs/vets-api/blob/43af9aeeee088347cbdce6a8797fb213ac112548/spec/factories/iam_users.rb#L40

https://github.com/department-of-veterans-affairs/vets-api/blob/43af9aeeee088347cbdce6a8797fb213ac112548/spec/lib/mpi/responses/add_parser_spec.rb#L34

I know I saw it in LH docs somewhere too, let me dig it up.

Reply:
[This](https://github.com/department-of-veterans-affairs/vets-api/blame/master/lib/simple_forms_api_submission/metadata_validator.rb#L33) is where it throws an error if it is not (although, admittedly this is in another, non-lighthouse, teams code. So... not positive this can be definitive proof)

Reply:
![Screenshot 2024-10-28 at 12 49 12 PM](https://github.com/user-attachments/assets/87c15ea8-5c34-42bc-b991-925ddf8967f8)

Reply:
good enough to me